io/module_wrt_grid_comp.F90

```diff
@@ -4536,16 +4537,16 @@ subroutine splat8(idrt,jmax,aslat)
 ! - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
      end subroutine splat8
 
-   !> ???
+   !> Compute rotated lat/lon grid
    !>
-   !> @param[in] tlmd ???
-   !> @param[in] tphd ???
-   !> @param[in] almd ???
-   !> @param[in] aphd ???
-   !> @param[in] tlm0d ???
-   !> @param[in] tph0d ???
+   !> @param[in] tlmd Longitude
+   !> @param[in] tphd Latitude
+   !> @param[in] almd Rotated longitude
+   !> @param[in] aphd Rotated latitude
+   !> @param[in] tlm0d Central longitude
+   !> @param[in] tph0d Central latitude
```

Comment:
First two arguments are actually Rotated longitude and Rotated latitude.
Third and fourth arguments are longitude (or geographical) longitude and latitude. They are intent [out], not [in].